src/test/mediapipe/config_mediapipe_dummy_adapter_full_dummy_in_both_config_and_subconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "model_config_list": [
+        {"config": {
+            "name": "dummy",
+            "base_path": "/ovms/src/test/dummy",
+            "shape": "(1, 10)"
+    }
+    }
+    ],
+    "mediapipe_config_list": [
+    {
+        "name":"mediaDummyADAPTFULL",
+        "graph_path":"/ovms/src/test/mediapipe/graphdummyadapterfull.pbtxt",
+        "subconfig":"/ovms/src/test/mediapipe/config_standard_dummy.json"
+    },
+    {
+        "name":"mediaDummy",
```

Comment:
I assume we could change one of those graphs to use base_path without passing full graph_path/subconfig path when base_path support is added?